Component-Page/component-page.css

```diff
@@ -0,0 +1,48 @@
+h1.sub-heading {
+    font-size: 1.5rem;
+    margin: 1rem 0;
+}
+
+.demo-with-code {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    align-items: center;
+    margin-bottom: 3rem;
+}
+
+.demo-div {
+    display: flex;
+    flex-wrap: wrap;
+    background-color: #F7E800;
+    justify-content: center;
+    gap: 1.5rem;
+    align-items: center;
+    color: black;
+    border-radius: 5px;
+    padding: 20px;
+    width: 100%;
+    margin-bottom: 1rem;
+}
+
+.code-examples {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 5px;
+    width: 100%;
+    background-color: #282c34;
```

Comment:
Use css variables

Reply:
I've done this across the site wherever there were hex values used. Thanks for the suggestion, looks much better now.